README.md

```diff
@@ -0,0 +1,23 @@
+Dart Mode
+=========
+Dart Mode is a major mode for editing Dart files in Emacs.
+
+## Installation
```

Comment:
I think we should suggest installing via the Emacs package manager. People can get dart-mode using the instructions here: https://marmalade-repo.org/#download